incubator/hnc/api/v1alpha1/hnc_config_conversion.go

```diff
@@ -39,8 +41,13 @@ func (src *HNCConfiguration) ConvertTo(dstRaw conversion.Hub) error {
 	dstSpecTypes := []v1a2.TypeSynchronizationSpec{}
 	for _, st := range srcSpecTypes {
 		dt := v1a2.TypeSynchronizationSpec{}
-		dt.APIVersion = st.APIVersion
-		dt.Kind = st.Kind
+		// Hack the group from APIVersion by removing the version.
+		gv := strings.Split(st.APIVersion, "/")
+		if len(gv) == 2 {
+			dt.Group = gv[0]
+		}
+		// Hack the resource from Kind by using the lower case.
+		dt.Resource = strings.ToLower(st.Kind) + "s"
```

Comment:
Yes, otherwise singles will be considered as not found. What about having a specific case for `networkpolicies` mapping?

Reply:
I mean, anything that ends in "-y" could be replaced by "-ies." That's the standard English pattern and the two of those should catch ~99% of all cases, I think.

What happens if we convert something incorrectly?

Reply:
it will have status.conditions `TypeNotFound` and not listed in status.types. See https://github.com/kubernetes-sigs/multi-tenancy/pull/1194#discussion_r506473410 for the full spec and status for wrong `networkpolicys` example for now.

SG will do `y` to `ies` mapping!